cocos/ui/scroll-bar.ts

```diff
@@ -41,11 +41,13 @@ import { Node } from '../core';
 import { legacyCC } from '../core/global-exports';
 
 const GETTING_SHORTER_FACTOR = 20;
-const ZERO = new Vec3();
+const _zeroVec2 = new Vec2();
```

Comment:
If we expect it to remain unchanged, you should use Vec2.ZERO directly, no need to create an object here